tests/drivers/at/Makefile

```diff
@@ -2,8 +2,11 @@ include ../Makefile.drivers_common
 
 USEMODULE += shell
 USEMODULE += at
-USEMODULE += at_urc_isr_medium
```

Comment:
Async URC handling is broken. See [this issue](https://github.com/RIOT-OS/RIOT/issues/20059).